test/specs/lwcLsp.e2e.ts

```diff
@@ -55,7 +55,7 @@ describe('LWC LSP', async () => {
     await utilities.pause(1);
 
     // Verify 'Go to definition' took us to the definition file
-    const editorView = await workbench.getEditorView();
+    const editorView = workbench.getEditorView();
```

Comment:
nice catch! Could you also remove that await in other places this function is called?